lib/rubocop/cop/sorbet/forbid_untyped_struct_props.rb

```diff
@@ -47,19 +42,19 @@ class ForbidUntypedStructProps < RuboCop::Cop::Base
 
         # @!method subclass_of_t_struct?(node)
         def_node_matcher :subclass_of_t_struct?, <<~PATTERN
-          (class (const ...) {#t_struct #t_immutable_struct} ...)
+          (class (const ...) #t_struct ...)
         PATTERN
 
         # @!method untyped_props(node)
```

Comment:
@paracycle what do you think of this (RE: https://github.com/Shopify/rubocop-sorbet/pull/148/files#r1113271304)

```suggestion
        # @!method untyped_props(node)
        # Search for untyped prop/const declarations and capture their types
```

Reply:
Ok, that's fair. Let's go with your suggested change.

Reply:
Okay, sounds good. I've opened https://github.com/Shopify/rubocop-sorbet/pull/155 (although its currently empty) for us to fill in contributing documentation, where we can link to the node pattern docs as something good to know before getting started.